pkg/apis/complianceoperator/v1alpha1/compliancescan_types.go

```diff
@@ -37,10 +37,22 @@ const (
 // ComplianceScanSpec defines the desired state of ComplianceScan
 // +k8s:openapi-gen=true
 type ComplianceScanSpec struct {
-	ContentImage string            `json:"contentImage,omitempty"`
-	Profile      string            `json:"profile,omitempty"`
-	Rule         string            `json:"rule,omitempty"`
-	Content      string            `json:"content,omitempty"`
+	// Is the image with the content (Data Stream), that will be used to run
+	// OpenSCAP.
+	ContentImage string `json:"contentImage,omitempty"`
+	// Is the profile in the data stream to be used. This is the collection of
+	// rules that will be checked for.
+	Profile string `json:"profile,omitempty"`
+	// A Rule can be specified if the scan should check only for a specific
+	// rule. Note that when leaving this empty, the scan will check for all the
+	// rules for a specific profile.
+	Rule string `json:"rule,omitempty"`
+	// Is the path to the file that contains the content (the data stream)
```

Comment:
Should we say that the init container by default copies the content to `/content`? Should we say that if the path is relative, then the scan pod would produce a path based at `/content`?

Reply:
Yes, we should mention the specifics of this. Specially if we have requirements. IMO we don't need to mention anything about the `/content` directory, but we should instead mention that the path is relative to the container's `/` directory.

Reply:
OK, that works for me. I was actually thinking that we might want to drop the `if path is relative then path := /content+path` logic as it might be confusing for people unless it's documented.

Reply:
Resolved in PR #151